src/_posts/databases/2000-01-01-backup-policies.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Databases Backup Policies
+nav: Backup Policies
+modified_at: 2024-11-28 12:00:00
+tags: databases backup policy policies
+index: 2
+---
+
+
+Scalingo offers several options for backing up and restoring your data,
+depending on the service used. All backups are encrypted and securely stored in
```

Comment:
```suggestion
depending on the database used. All backups are encrypted and securely stored in
```
service => database? I think it's more clear

Reply:
I put "depending on the database type and on the plan used"